hbase-server/src/test/java/org/apache/hadoop/hbase/master/region/MasterRegionTestBase.java

```diff
@@ -78,6 +78,11 @@ public void setUp() throws IOException {
     htu.getConfiguration().setBoolean(MemStoreLAB.USEMSLAB_KEY, false);
     // Runs on local filesystem. Test does not need sync. Turn off checks.
     htu.getConfiguration().setBoolean(CommonFSUtils.UNSAFE_STREAM_CAPABILITY_ENFORCE, false);
+
+    create();
+  }
+
+  public void create() throws IOException {
```

Comment:
Better give it a more specific name?  And does it need to be public? Protected is enough?